.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,90 @@
+name: Bug report
+description: File a bug report
+title: '[Bug]: '
+labels: ['bug']
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: How to reproduce
+      description: Steps to reproduce the behavior (code, online sandbox, example repository)
```

I'd remove the sandbox since I haven't been able to make it fully working there yet. [Here](https://codesandbox.io/p/devbox/tuono-7qx7gk) the codesandbox experiment I did.
 
```suggestion
      description: Steps to reproduce the behavior (code, example repository)
```